aten/src/ATen/native/cuda/cutlass_extensions/epilogue_helpers.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file epilogue_helpers.h
+ *
+ * This file includes types for the epilogues. The empty structs exist so we can signal to template
+ * code the type of epilogue we want to run, and let the underlying code specify the details such as
+ * element types, accumulator type and elements per vector access.
+ *
+ */
+
+#pragma once
+
+#include "cutlass/epilogue/thread/linear_combination.h"
```

Comment:
Unfortunately all of these includes will need to use "<" and ">" instead of quotes.

Reply:
No problem, these files have to go through the linter anyway, it will fix it.  I was just intending to do it just before the merge, so that I could save their current versions at that point, namely I had to made some changes vs. originals copied from FasterTransfomer (for update to CUTLASS 3.x, etc.), but once they linted according to the PyTorch lint rules, there will be many more differences.

Reply:
Yeah since we're including these files directly I don't think we can dodge these constraints.